### DISCUSSION TITLE
[Suggestion] Add Fade Anim to Lifebar

### DISCUSSION THREAD

**Discussion by doomjoshuaboy** [ikemen-engine/Ikemen-GO#867] [Feature Request] [answered]
hi guys since you had fade in and out for lifebars So an suggestion came up on my mind.

https://user-images.githubusercontent.com/22404668/124854357-8dd7bf00-dfea-11eb-9472-68628fae6c11.mp4

so in rounds section I would like this to be a thing.
`fadein.anim`
`fadeout.anim `

This is how it looks in Fighter factory 3 when i added them in there:
![ff3_DP3nCcBdAv](https://user-images.githubusercontent.com/22404668/124881826-1ff3bd80-e013-11eb-8da1-81e920ff357e.png)

This is when I added in and it didn't happen 
https://streamable.com/ietlay

This is what I am trying to add into the lifebar (I know its kinda fast as i have no clue on making it slower)   
![ezgif com-gif-maker](https://user-images.githubusercontent.com/22404668/124887778-d27a4f00-e018-11eb-8e33-62dbfd0f5a8c.gif)
[lifebartest.zip](https://github.com/Windblade-GR01/Ikemen-GO/files/6783002/lifebartest.zip)

**Comment by K4thos** [comment, marked as answer]
it already works in nightly
